backend/api_handler.py

```diff
@@ -12,7 +12,9 @@
 from dataall.api.Objects import bootstrap as bootstrap_schema, get_executable_schema
```

Comment:
we may think a little bit about the folder structure since we have a back-end then inside we have a data.all, isn't the full project data.all specific? looks like api_handler.py, aws_handler.py, cdkproxymain.py by its folder structure are not data.all specific? Similar applies to docker/prod, migrations

Reply:
I can only try to speculate why it was done this way. 

I think that we can move all handlers to be part of `dataall` package, but they should be treated very carefully since there are docker and docker-compose files that depend on them as a entrypoint. And we should do it in a dedicated PR, just in case. 
About docker and migrations, I don't think that we should move them at all. Since `dataall` is a python package we don't want to import a migration somehow from the code. Let better them stay remote)